### PR TITLE
128 improvement of the readme file

### PR DESCRIPTION
**Pull request by Pelayo-Reguera**
Improvement of all the README files in the repository.

### CODE REVIEW

**Review by UO283069 — APPROVED** [review]

File: restapi/README.md [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
-- express-prom-bundle and prom-client: middleware to capture the petitions and send statistics to prometheus. This is useful for monitoring the webservice.
-- @types packages. Types for coding using typescript.
+# Restapi
+The purpose for this part is to be a rest API using Express and Node.js using TypeScript, for 
+all the actions involving the centralised databased MongoDB :leaves: which is in charge of 
+storing the minimal information of public things. The 
+webservice will be deployed using Docker :whale2:.
      
-The code is quite straight forward, the [server.ts](server.ts) file launchs the api. The [api.ts](api.ts) is actually the api, where you will see there two api entry points, one post for creating a new user, and one get to list all the users. 
+The code is quite straight forward, the [server.ts](server.ts) file launchs the api. The [api.ts]
```
Comment (uo282892) [inline restapi/README.md:7]: Typo: It is "launches"

Comment (uo282892) [reply restapi/README.md:7]: Also, the api.ts link does not work here

**Review by uo282892 — COMMENTED** [review]
Check the typos in the readme of restapi

**Review by uo282892 — APPROVED** [review]

**Review by UO283428 — APPROVED** [review]

**Review by Davidmc07 — CHANGES_REQUESTED** [review]
I would use as logo the favicon or the custom placemark image

**Review by Davidmc07 — APPROVED** [review]

**Pull request merged**